provide-storage/backup-storage-provider/bsp-cli-flags.md

```diff
@@ -0,0 +1,92 @@
+---
+title: BSP CLI Flags
+description: Configuration flags for running a Backup Storage Provider node, covering capacity limits, task execution, proof handling, and operational tuning.
+---
+
+# BSP CLI Flags
+
+This page documents all CLI flags available when running a Backup Storage Provider (BSP) node, including storage capacity, task execution, and operational behavior.
+
+## Core Provider Flags
+
+Core provider flags are required to run a node in BSP mode. They help define storage capacity, provider identity, and local storage backend configuration.
+
+| Flag | Description | Required | Default |
+|:------|:-------------|:----------|:---------|
+| `--provider` | Enable storage provider mode | Yes | false |
+| `--provider-type bsp` | Set provider type to BSP | Yes | None |
+| `--max-storage-capacity <BYTES>` | Maximum storage capacity | Yes | None |
+| `--jump-capacity <BYTES>` | Jump capacity for new storage | Yes | None |
+| `--storage-layer <TYPE>` | Storage backend (`rocksdb` or `memory`) | No | `memory` |
+| `--storage-path <PATH>` | Storage path (required if rocksdb) | No | None |
+
+**Example Values:**
+- `--max-storage-capacity 858993459200` (800 GiB = 80% of 1 TB disk)
```

Comment:
This bullet point got messed up somehow
<img width="755" height="223" alt="Screenshot 2025-12-29 at 6 10 53 PM" src="https://github.com/user-attachments/assets/55f3730a-cdd3-405d-b76a-fb3693f06644" />